scripts/solve_network.py

```diff
@@ -477,37 +476,47 @@ def extra_functionality(n, snapshots):
     add_co2_sequestration_limit(n, snapshots)
 
 
-def solve_network(n, config, opts="", **kwargs):
-    solver_options = config["solving"]["solver"].copy()
-    solver_name = solver_options.pop("name")
-    cf_solving = config["solving"]["options"]
-    track_iterations = cf_solving.get("track_iterations", False)
-    min_iterations = cf_solving.get("min_iterations", 4)
-    max_iterations = cf_solving.get("max_iterations", 6)
+def solve_network(n, config, solving, **kwargs):
+    set_of_options = solving["solver"]["options"]
+    cf_solving = solving["options"]
+
+    kwargs["solver_options"] = (
+        solving["solver_options"][set_of_options] if set_of_options else {}
+    )
+    kwargs["solver_name"] = solving["solver"]["name"]
+    kwargs["extra_functionality"] = extra_functionality
+
+    if kwargs["solver_name"] == "gurobi":
+        logging.getLogger("gurobipy").setLevel(logging.CRITICAL)
+    rolling_horizon = cf_solving.pop("rolling_horizon", False)

```

Comment:
Not sure we have an option for the rolling horizon... Agree that it may be interesting to advance implementation, but it feels like an issue per se. What do you think @GbotemiB?

Reply:
Yeah, we currently don't have an implementation for rolling horizon. But, I picked it from pypsa-eur. We can probably remove it for now and include it for advance implementation.

Reply:
Yeah. We don't have an implementation for `rolling horizon`. We can simply just leave it out and include it for an advance implementation.